vm/src/frame.rs

```diff
@@ -111,6 +111,7 @@ pub struct Frame {
     /// tracer function for this frame (usually is None)
     pub trace: PyMutex<PyObjectRef>,
     state: PyMutex<FrameState>,
+    pub temperary_refs: PyMutex<Vec<PyObjectRef>>,
```

Comment:
Maybe we can reuse this for `del`.
I didn't know many fields in Frame are mutex. I thought they are single-thread accessible.

typo
```suggestion
    pub temporary_refs: PyMutex<Vec<PyObjectRef>>,
```

Reply:
If frame itself is executed single-threaded, we could replace mutex with an atomic increasing vec, do we have any crates doing that?

Reply:
I don't think we have those crate. When we create safety by human, `UnsafeCell` is the common tool